python/common/python/math.java

```diff
@@ -0,0 +1,20 @@
+package python;
+
+@org.python.Module()
```

Comment:
Add module documentation, can be found in commit 4f16cbc883993ef5eacab7a492bd5774135e29e8 or 2c5d6c6fa78330dbd70b32ac4f5daceb518e1a2d .

Reply:
Fixed by db590ba2703eed682f3912b630c992379d0022bd